addons/binding/org.openhab.binding.netatmo/README.md

```diff
@@ -92,6 +92,10 @@ Number Netatmo_Indoor_CO2 "CO2" <carbondioxide> { channel = "netatmo:NAMain:home
 * Location
 * TimeStamp
 * LastStatusStore
+* MinTemp
+* MaxTemp
```

Comment:
Could you please document either in README.md or in the channel description what is the period for min and max temperatures ?